tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+skipsdist=True
+envlist = clean,ansible{2.12,2.13}-py{38,39,310}-{with_constraints,without_constraints},linters
+
+[testenv]
+description = Run the test-suite and generate a HTML coverage report
+deps =
+  pytest
+  pytest-cov
+  ansible2.12: ansible-core>2.12,<2.13
+  ansible2.13: ansible-core>2.13,<2.14
+  !ansible2.12-!ansible2.13: ansible-core
+  pytest-ansible-units
+
+[testenv:clean]
+deps = coverage
+skip_install = true
+commands = coverage erase
+
+[testenv:complexity-report]
+description = Generate a HTML complexity report in the complexity directory
+deps =
+  # See: https://github.com/lordmauve/flake8-html/issues/30
+  flake8>=3.3.0,<5.0.0'
+  flake8-html
+commands = -flake8 --select C90 --max-complexity 10 --format=html --htmldir={posargs:complexity} plugins
+
+[testenv:black]
+deps =
+  black==22.3.0
+commands =
+  black -v -l79 .
```

Comment:
Please, drop the `-l79`. Otherwise we will get a non standard back formatting which will be a source of problem later. I don't think anyone have a problem with lines slightly longer than 79 characters.